src/client/game.cpp

```diff
@@ -2423,41 +2404,29 @@ void Game::updateCameraOrientation(CameraOrientation *cam, float dtime)
 
 void Game::updatePlayerControl(const CameraOrientation &cam)
 {
-	//TimeTaker tt("update player control", NULL, PRECISION_NANO);
+	LocalPlayer *player = client->getEnv().getLocalPlayer();
 
-	// DO NOT use the isKeyDown method for the forward, backward, left, right
-	// buttons, as the code that uses the controls needs to be able to
-	// distinguish between the two in order to know when to use joysticks.
+	//TimeTaker tt("update player control", NULL, PRECISION_NANO);
 
 	PlayerControl control(
-		input->isKeyDown(KeyType::FORWARD),
-		input->isKeyDown(KeyType::BACKWARD),
-		input->isKeyDown(KeyType::LEFT),
-		input->isKeyDown(KeyType::RIGHT),
-		isKeyDown(KeyType::JUMP),
-		isKeyDown(KeyType::AUX1),
-		isKeyDown(KeyType::SNEAK),
-		isKeyDown(KeyType::ZOOM),
-		isKeyDown(KeyType::DIG),
-		isKeyDown(KeyType::PLACE),
+		input->isKeyDown(KeyType::JUMP) || player->getAutojump(),
```

Comment:
Is `player->getAutojump()` intentional? Why is it required?

Reply:
Thanks for the review, I'll have a look if this was in the original PR or is an error by my rebase

Reply:
it's in the original PR, and it is needed. If I remove the `|| player->getAutojump()` part, then the auto-jump doesn't work anymore